lob/resource.py

```diff
@@ -64,9 +64,6 @@ def __getattr__(self, k):
         except KeyError:
             raise AttributeError(k) #pragma: no cover
 
-    def __setattr__(self, k, v):
-        self[k] = v
```

Comment:
Interestingly enough, I don't think we ever needed this in the first place, since `LobObject` inherits from `dict` and this is probably how `dict` sets attributes anyway